code/Items/Data/ItemData.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace ssl.Items.Data
+{
+    public abstract class ItemData
+    {
+        public ItemData(string id, string name, string model)
+        {
+            Id = id;
+            Name = name;
+            Model = model;
+        }
+        
+        public string Id { get; }
+        public string Name { get; }
+        public string Model { get; } = ""; //Find default model
+
+        public abstract Item create();
```

Comment:
Create.
It can be virtual. Because some Items have a basic ItemData i think.
I think we have to let the game create an Item (= item without any specification) and an ItemData define the item